### PR TITLE
DOCS-953 JDBC 42.7.3.2

### PR DESCRIPTION
**Pull request by jpe442**
Updating docs for new version of JDBC connector 42.7.3.2

### CODE REVIEW

**Review by ericmack — APPROVED** [review]
Thanks

File: product_docs/docs/jdbc_connector/42.7.3.2/01_jdbc_rel_notes/jdbc_42.7.3.2_rel_notes.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+| Enhancement         | Added support for EDB Postgres Advanced Server 17.2                                                                                                                                                                                                                               |                                                                                                                                                                                                                                                                                                                                                                          
+| Fix                 | The JDK 8 compatibility is broken in ver 42.7.3.1 of edb-jdbc18 driver.                                                                                                                                                                                                           |
```
Comment (gvasquezvargas) [inline product_docs/docs/jdbc_connector/42.7.3.2/01_jdbc_rel_notes/jdbc_42.7.3.2_rel_notes.mdx:16]: Usually, we distinguish them between security fixes or bug fixes. I would suggest going over the list and checking which could be bugs or security to categorize them that way? 

Also, there is a style/way in which we phrase the notes. The [Release Notes Style Guide](https://www.enterprisedb.com/docs/community/contributing/releasenotes-styleguide/) has some examples. I am changing this bullet here as I would to it, but I leave it up to you whether you want to do the adaptations that way or not :) 
```suggestion
| Bug fix                 | Fixes an incompatibility issue with JDK 8 that found in version 42.7.3.1 of the `edb-jdbc18` driver.                                                                                                                                                                                                           |                                                   
```

File: product_docs/docs/jdbc_connector/42.7.3.2/01_jdbc_rel_notes/jdbc_42.7.3.2_rel_notes.mdx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+| Type                | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
+|---------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Performance         | Improved parsing performance with large SQL (MTK/SQL Plus)                                                                                                                                                                                                                        |
```
Comment (gvasquezvargas) [inline product_docs/docs/jdbc_connector/42.7.3.2/01_jdbc_rel_notes/jdbc_42.7.3.2_rel_notes.mdx:14]:
```suggestion
| Performance         | Improved parsing performance with large SQL (MTK/SQL Plus).                                                                                                                                                                                                                        |                                                                                                                                                                                                                                                           
```

File: product_docs/docs/jdbc_connector/42.7.3.2/01_jdbc_rel_notes/jdbc_42.7.3.2_rel_notes.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+|---------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Performance         | Improved parsing performance with large SQL (MTK/SQL Plus)                                                                                                                                                                                                                        |                                                                                                                                                                                                                                                           
+| Enhancement         | Added support for EDB Postgres Advanced Server 17.2                                                                                                                                                                                                                               |
```
Comment (gvasquezvargas) [inline product_docs/docs/jdbc_connector/42.7.3.2/01_jdbc_rel_notes/jdbc_42.7.3.2_rel_notes.mdx:15]:
```suggestion
| Enhancement         | Added support for EDB Postgres Advanced Server 17.2.                                                                                                                                                                                                                               |                                                                                                                                                                                                                                                                                                                                                                          
```

File: product_docs/docs/jdbc_connector/42.7.3.2/01_jdbc_rel_notes/jdbc_42.7.3.2_rel_notes.mdx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+| Fix                 | edb-jdbc installation should not install a lower JDK version when a higher version is installed.                                                                                                                                                                                  |                                                                                                                                                                                                                                                                                                                                                       
+| Fix                 | Fixed issue where Message.getJMSMessageID() returns null.                                                                                                                                                                                                                         |
```
Comment (gvasquezvargas) [inline product_docs/docs/jdbc_connector/42.7.3.2/01_jdbc_rel_notes/jdbc_42.7.3.2_rel_notes.mdx:18]: I would format some of these objects, maybe something like this? 
```suggestion
| Fix                 | Fixed issue where `Message.getJMSMessageID()` returns `null`.                                                                                                                                                                                                                         |                                                                                                                                                                                                                                                                                                                                                 
```

File: product_docs/docs/jdbc_connector/42.7.3.2/01_jdbc_rel_notes/jdbc_42.7.3.2_rel_notes.mdx [hunk 5]
```diff
@@ -0,0 +1,33 @@
+---
+title: "EDB JDBC Connector 42.7.3.2 release notes"
+navTitle: Version 42.7.3.2
+---
+
+Released: 20 Nov 2024
```
Comment (gvasquezvargas) [inline product_docs/docs/jdbc_connector/42.7.3.2/01_jdbc_rel_notes/jdbc_42.7.3.2_rel_notes.mdx:6]: I believe the date is 21. but let's see what Eric says
```suggestion
Released: 21 Nov 2024
```

**Review by gvasquezvargas — APPROVED** [review]
Left some suggestions, otherwise LGTM.

File: product_docs/docs/jdbc_connector/42.7.3.2/01_jdbc_rel_notes/jdbc_42.7.3.2_rel_notes.mdx (hunk 5 above)
Comment (gvasquezvargas) [reply product_docs/docs/jdbc_connector/42.7.3.2/01_jdbc_rel_notes/jdbc_42.7.3.2_rel_notes.mdx:6]: note: if this changes, we'll have to update the index too.

File: product_docs/docs/jdbc_connector/42.7.3.2/01_jdbc_rel_notes/jdbc_42.7.3.2_rel_notes.mdx [hunk 6]
```diff
@@ -0,0 +1,33 @@
+---
+title: "EDB JDBC Connector 42.7.3.2 release notes"
+navTitle: Version 42.7.3.2
+---
+
+Released: 21 Nov 2024
```
Comment (gvasquezvargas) [inline product_docs/docs/jdbc_connector/42.7.3.2/01_jdbc_rel_notes/jdbc_42.7.3.2_rel_notes.mdx:6]:
```suggestion
Released: 22 Nov 2024
```

File: product_docs/docs/jdbc_connector/42.7.3.2/01_jdbc_rel_notes/index.mdx [hunk 7]
```diff
@@ -15,6 +16,7 @@ These release notes describe what's new in each release. When a minor or patch r
 
 | Version                                  | Release Date | 
 | ---------------------------------------- | ------------ | 
+| [42.7.3.2](jdbc_42.7.3.2_rel_notes)      | 20 Nov 2024 |
```
Comment (gvasquezvargas) [inline product_docs/docs/jdbc_connector/42.7.3.2/01_jdbc_rel_notes/index.mdx:19]:
```suggestion
| [42.7.3.2](jdbc_42.7.3.2_rel_notes)      | 21 Nov 2024 |
```

**Pull request merged**